pkgs/development/python-modules/pyqt/6.x.nix

```diff
@@ -55,8 +54,10 @@ buildPythonPackage rec {
     verbose = true
     EOF
 
+    # pythonRelaxDeps doesn't work and the wanted versions are not released AFAIK
     substituteInPlace pyproject.toml \
-      --replace-fail 'version = "${version}"' 'version = "${lib.versions.pad 3 version}"'
```

Comment:
Why is this no longer necessary?

Reply:
It failed to build when leaving it in place and builds without it so I assumed it was fine to remove.

I'm mainly doing this PR to try and un-block master and https://github.com/NixOS/nixpkgs/pull/348417. If anyone is more experienced with python packages I would be happy to hand over the reigns since I don't really know what I'm doing.

Reply:
I tried running nixpkgs-review, and the resulting version number is not accepted by multiple dependent packages, we should keep the patching and fix it.

Reply:
I added it back and now it seems to work, it was likely me making some mistake before. I pushed the commit with it re-added.